core/src/main/resources/org/apache/spark/ui/static/stagepage.js

```diff
@@ -286,7 +286,7 @@ $(document).ready(function () {
         " Show Additional Metrics" +
         "</a></div>" +
         "<div class='container-fluid container-fluid-div' id='toggle-metrics' hidden>" +
-        "<div><input type='checkbox' class='toggle-vis' id='box-0' data-column='0'> Select All</div>" +
+        "<div id='select_all' class='select-all-checkbox-div'><input type='checkbox' class='toggle-vis' id='box-0' data-column='0'> Select All</div>" +
```

Comment:
Just curious, what is the use of `id` and `class` here? How was it working fine without `id` and `class`? Does it take some default value?

Reply:
`class` is to specify a width of `Select All` checkbox.
`id` is not used currently but I put it  just in case as well as any other checkbox.

Reply:
Looks logical, but I'm wondering why the other browsers(Chrome/Safari) show correctly without that information? Do you know Chrome also has this issue that `Pop!_OS` because they might share WebKit?